profiles/pp/modules/custom/user_profile_import/user_profile_import.routing.yml

```diff
@@ -1,18 +1,15 @@
-# Ajax page for import specific user
-
 user_profile_import_arguments:
-  path: 'import_profiles/{page}'
+  path: '/import_profiles/{page}'
   defaults:
-    _content: '\Drupal\user_profile_import\Controller\UserProfileImportController::importProfiles'
+    _controller: 'Drupal\user_profile_import\Controller\UserProfileImportController::importProfiles'
+    _title: 'Import in progress'
   requirements:
     _access: 'TRUE'
```

Comment:
Does this mean we introducing security hole? Everybody will may to use this route.
Let's get protected